examples/nextjs-trpc/src/trpc/react.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import { QueryClientProvider, type QueryClient } from "@tanstack/react-query";
+import { loggerLink, unstable_httpBatchStreamLink } from "@trpc/client";
+import { createTRPCReact } from "@trpc/react-query";
+import { type inferRouterInputs, type inferRouterOutputs } from "@trpc/server";
```

Comment:
```suggestion
import type { inferRouterInputs, inferRouterOutputs } from "@trpc/server";
```
Just some nice cleanup